packages/lit-dev-content/samples/tutorials/wc-to-lit/tutorial.json

```diff
@@ -0,0 +1,56 @@
+{
+  "header": "From Web Component to Lit Element",
+  "difficulty": "Beginner",
+  "size": "medium",
+  "duration": 43,
+  "category": "Build",
+  "imgSrc": "images/tutorials/wc-to-lit/wclogo.png",
+  "imgAlt": "The Web Components logo",
+  "steps": [
+      {
+        "title": "Introduction",
+        "hasAfter": true
+      },
+      {
+        "title": "Define A Custom Element"
+      },
+      {
+        "title": "Custom Element Lifecycle & Rendering"
+      },
+      {
+        "title": "Shadow DOM"
+      },
+      {
+        "title": "HTML Templates"
+      },
+      {
+        "title": "Adding Functionality - Properties"
+      },
+      {
+        "title": "Adding Functionality - Attributes"
+      },
+      {
+        "title": "Adding Functionality - Attribute Styling"
+      },
+      {
+        "title": "Adding Functionality - Listeners"
+      },
+      {
+        "title": "lit-html"
+      },
+      {
+        "title": "lit-html - Declarative Templating"
```

Comment:
Updating these to match the current content of the steps. 

The rest of the docs use sentence case for headings & page titles--like I've used here. We should probably be consistent throughout, but I haven't made this change elsewhere because it's a lot.